regolith/helpers/a_proposalhelper.py

```diff
@@ -148,73 +127,65 @@ def db_updater(self):
             else:
                 pdoc.update({'authors': rc.authors})
         else:
-            pdoc.update({'authors': 'tbd'})
-        pdoc.update({
-            'begin_day': rc.begin_day,
-            'begin_month': rc.begin_month,
-            'begin_year': rc.begin_year,
-            'call_for_proposals': rc.call_for_proposals,
-            # 'cpp_info':
-        })
+            pdoc.update({'authors': ['tbd']})
+        if rc.begin_date:
+            pdoc.update({'begin_date': rc.begin_date})
+        cpp_info = {}
+        if rc.cpp_flag:
+            cpp_info['cppflag'] = rc.cpp_flag
+        if rc.cpp_agencies:
+            cpp_info['other_agencies_submitted'] = rc.cpp_agencies
+        if rc.cpp_institution:
+            cpp_info['institution'] = rc.cpp_institution
+        if rc.rc.cpp_months_academic:
+             cpp_info['person_months_academic'] = rc.cpp_months_academic
+        if rc.cpp_months_summer:
+            cpp_info['person_months_summer'] =  rc.cpp_months_summer
+        if rc.cpp_scope:
+            cpp_info['project_scope'] = rc.cpp_scope
+        if cpp_info:
+            pdoc.update({'cpp_info':cpp_info})
         if rc.currency:
             pdoc.update({'currency': rc.currency})
         else:
             pdoc.update({'currency': 'USD'})
-        pdoc.update({'day': now.day
-                     'due_date': rc.due_date
-                     })
+        if rc.submitted_date:
+            pdoc.update({'submitted_date': submitted_date})
+        else:
+            pdoc.update({'submitted_date': 'tbd'})
+        if rc.due_date:
+            pdoc.update({'due_date': rc.due_date})
+        else:
+            pdoc.update({'due_date': 'tbd'})
         if rc.duration:
-            pdoc.update('duration': rc.duration)
+            pdoc.update({'duration': rc.duration})
         else:
             pdoc.update({'duration': 'tbd'})
-        pdoc.update({
-            'end_day': rc.end_day,
-            'end_month': rc.end_month,
-            'end_year': rc.end_year,
-            'funder': rc.funder,
-            'full': rc.full
-            'month': now.month # convert numeric to string
-            'notes': rc.notes
-        })
+        if rc.end_date:
+            pdoc.update({'end_date': rc.end_date})
+        if rc.funder:
+            pdoc.update({'funder': rc.funder})
+        if rc.full:
+            pdoc.update({'full': rc.full})
+        if rc.notes:
+            pdoc.update({'notes': rc.notes})
         if rc.pi:
             pdoc.update({'pi': rc.pi})
         else:
             pdoc.update({'pi': 'tbd'})
-        if rc.status:
-            pdoc.update({'status': rc.status})
-        else:
-            pdoc.update({'status': 'tbd'})
-        # pdoc.update({'team':})
+        pdoc.update({'status': 'inprep'})
+        sample_team = {'cv':'http://pdf.com/goodbye-cv',
```

Comment:
no magic please, just empty list for team if not specified

Reply:
I used this as a "sample team member", sort of like the sample milestone in the projectum adder, which does not allow the user to pass in a milestone but instead provides a template that can be edited and replicated. I will change it to an empty list. Do you think I should allow the user to pass in a list of team members, each with their {cv, name, email, institution, name, position, subaward_amount} ? The way I can think of going about this is either
a) define arguments `member1_cv,` `member2_cv,...` up to `member10_cv` and so on for `name,` `email,` `institution...`
b) allow the user to pass in a list of `cv`'s, a list of `name`s, etc. for the team members and then construct a member dictionary by iterating through these lists, i.e. `cv[0],` `names[0],` etc. would be assigned to the first member, `cv[1],` `names[1],` etc. to the second and so on. But this method can be problematic when the lists for each key type are not aligned, for e.g. member 1 specifies a `cv[0]` but member 2 does not, and `cv[1]` is actually the `cv` of member 3. So we would instruct the user to specify `None` for no cv 
I'm starting to think it might be easier for the user to simply edit the `proposals.yaml` file themselves for the `team` key!

Reply:
thanks.  Agile says no for now (but be careful, I may be requesting it when I decide I really need it!)